src/components/pages/SearchResultsPage.js

```diff
@@ -53,19 +74,30 @@ const SearchResultsPage = () => {
             <h1 className="text-white font-bold text-5xl">
               {location.state.value}
             </h1>
-            <h2 className="text-white font-semibold text-3xl mt-2">$138.08</h2>
+            <h2 className="text-white font-semibold text-3xl mt-2">
+              ${stockData[0].recentPrice}
+            </h2>
             <p className="text-bright-green font-medium text-md mt-2">
-              +$4.61 (+3.22%)
-              <p className="inline text-light-gray font-light"> Today</p>
+              {stockData[0].recentPrice - stockData[0].open >= 0 ? "+" : "-"} $
+              {Math.abs(stockData[0].recentPrice - stockData[0].open).toFixed(
+                2
+              )}{" "}
+              (
+              {(
+                ((stockData[0].recentPrice - stockData[0].open) /
+                  stockData[0].open) *
+                100
+              ).toFixed(2)}
+              %)<p className="inline text-light-gray font-light"> Today</p>
```

Comment:
This will not always be `Today` , the `percent change` will be based on the selected time frame. For example , 5 Day would be the percent change over the 5 day period. You can refer to Mobile for an example